src/status_im/contexts/wallet/events.cljs

```diff
@@ -99,20 +99,20 @@
  (fn [_ [toast-message]]
    {:fx [[:dispatch [:toasts/upsert {:type :positive :text toast-message}]]]}))
 
-(rf/reg-event-fx
- :wallet/remove-account-success
- (fn [_ [toast-message _]]
-   {:fx [[:dispatch [:wallet/get-accounts]]
-         [:dispatch [:wallet/get-keypairs]]
-         [:dispatch-later
-          {:ms       100
-           :dispatch [:hide-bottom-sheet]}]
-         [:dispatch-later
-          {:ms       100
-           :dispatch [:pop-to-root :shell-stack]}]
-         [:dispatch-later
-          {:ms       100
-           :dispatch [:wallet/show-account-deleted-toast toast-message]}]]}))
+(defn remove-account-success
+  [_ [toast-message _]]
+  {:fx [[:dispatch [:wallet/clean-current-viewing-account]]
+        [:dispatch [:wallet/get-accounts]]
+        [:dispatch [:wallet/get-keypairs]]
+        [:dispatch-later
+         {:ms       100
+          :dispatch [:hide-bottom-sheet]}]
+        [:dispatch-later
+         {:ms       100
+          :dispatch [:pop-to-root :shell-stack]}]
+        [:dispatch-later
+         {:ms       100
+          :dispatch [:wallet/show-account-deleted-toast toast-message]}]]})
```

Comment:
Why was this changed to a regular function?

Also checked out the branch and the function doesn't seem to be used 🤔

Reply:
I suppose this is where it is supposed to be used:
https://github.com/status-im/status-mobile/blob/7efa1a0f793a7b8d1b519802214b0169be306f29/src/status_im/contexts/wallet/events.cljs#L123
 but it is still called like an event

Reply:
Oh I was testing something, should have reverted. Thanks @BalogunofAfrica!